specification/validated-relationships-service-api.yaml

```diff
@@ -279,6 +279,7 @@ paths:
               examples:  
                 internalServerError:
                   $ref: '#/components/examples/InternalServerError'
+
```

Comment:
Commenting here cus can't on  line 210. May want to consider a different operation outcome block to reference
`$ref: '#/components/schemas/OperationOutcome'` to link to as the errors will likely be different for this endpoint. Perhaps easier once you get into the detailed error handling though

Reply:
Suggest we leave for the moment once we get further into the detail